src/pivot-table/components/icons/styles.ts

```diff
@@ -0,0 +1,6 @@
+import { baseFlex } from "../shared-styles";
+
+export const headerPlusMinusIconWrapperStyle = {
+  width: 13,
```

Comment:
Maybe expose this pixel value as a constant and also consume it in the icon components?